elements/multicolumn/README.md

```diff
@@ -2,19 +2,34 @@ Multicolumns are the main elements to create complex layouts. They can have mult
 
 The arrays in the “cols” array represent the columns. The minimum number of columns is 2 and it can be grown to as many as you need. We support max. 5 in our EDMdesigner editor, but there is no limitation built in the generator.
 
-The columns can't have padding, margin, background and border properties, these properties need to be added to the elements inside the children array. 
+The columns can't have padding, margin and border properties, these properties need to be added to the elements inside the children array. [Read more on set these properies](../../property-groups)
+
+There are new features in the generator, now it is possible to:
+ - set background properties to the multicolumn and/or the columns. (Keep in mind if generating `VML background` is set to `true` and you set background image on both elements, on Outlook email clients only the multicolumns image will appearing. [Read more on VML background](../../generator-settings/VMLbackground.md)
+  - define a child element called `COL_SPACER` and with its `width` and `height` propery set a spacing around the columns inside the multicolumns. (it effects a more predictable and unified layout across various email clients)
 
 The width of the multicolumn is set to 100% and extends to the full width of its parent element. The width of the child elements of the columns is calculated from the width of the multicolumns parent elements width.
+
 Keep in mind that: 
- - the sum of the child elements width (with the padding, margin and border width values) won't exceed the parent elements width, because the last column might break into a new line
+ - the sum of the child elements width (with the padding, margin and border width values) won't exceed the parent elements width, because the last column might break into a new line (it applies to the column spacings too, the sum of the columns and column spacings width might not pass 100%)
  - too long words in a paragraph might push out the width of the column that result in breaking the layout
 
+Older JSON version:
```

Comment:
As far as I know the older JSON version did not have bg props ;) Also, instead of the stacking prop, we had reorderOnMobile, right?

Reply:
Ohh, the older does not mean the oldest I guess

Reply:
Anyways, it's easy to misunderstand.